src/opm/input/eclipse/Schedule/UDQ/UDQEnums.cpp

```diff
@@ -21,15 +21,44 @@
 
 #include <opm/common/utility/String.hpp>
 
+#include <charconv>
 #include <map>
+#include <optional>
+#include <regex>
 #include <set>
 #include <stdexcept>
 #include <string>
+#include <string_view>
+#include <system_error>
 #include <unordered_map>
 #include <vector>
 
 namespace {
 
+    std::string_view dequote(std::string_view s)
+    {
+        auto b = s.find_first_of("'");
```

Comment:
Looks a bit strange. Did you want to search for ' and " (`s.find_first_of("'\"")`), maybe?
If not then using find("'") and find("'", pos) without any substring creating might be better?

Reply:
> Did you want to search for `'` and `"` (`s.find_first_of("'\"")`), maybe?

No, not really.  I only want `'` here.  Double-quote (`"`) is not a valid input character.

> If not then using `find("'")` and `find("'", pos)` without any substring creating might be better?

Well, these are string views, not strings, so substrings are cheap.  On the other hand, `find_first_of()` isn't really needed here so I'll push an update to use the version below instead
```C++
std::string_view dequote(std::string_view s)
{
    const auto b = s.find('\'');
    if (b == std::string_view::npos) {
        return s;
    }

    const auto e = s.find('\'', b + 1);
    return { s.begin() + b + 1, e - b - 1 };
}
```

Reply:
> `find_first_of()` isn't really needed here so I'll push an update to use the version below instead

I've pushed that update now and I'll run another build check.